src/psyclone/tests/domain/lfric/lfric_integer_builtins_test.py

```diff
@@ -1408,14 +1417,17 @@ def test_int_inc_min_aX(tmpdir, monkeypatch, annexed, dist_mem):
 
 
 def test_real_X(tmpdir, monkeypatch, annexed, dist_mem):
-    ''' Test that 1) the str method of LFRicRealXKern returns the
-    expected string and 2) we generate correct code for the built-in
-    operation Y = real(X, r_def) where Y is a real-valued field, X is the
-    integer-valued field being converted and the correct kind, 'r_def',
-    is read from the PSyclone configuration file. Test with and without
-    annexed dofs being computed as this affects the generated code.
+    '''Test that 1) the str method of LFRicRealXKern returns the expected
+    string and 2) we generate correct code for the built-in operation
+    Y = real(X, r_def) where Y is a real-valued field, X is the
+    integer-valued field being converted and the correct kind,
+    'r_def', is read from the PSyclone configuration file. Test with
+    and without annexed dofs being computed as this affects the
+    generated code. 3: Also test the metadata() method.
```

Comment:
s/3:/3)/g

Reply:
Oops. Also added quotes around methods and classes in docstrings.

Reply:
Left-over `:`